src/Stratis.Bitcoin.Features.Api/ApiSettings.cs

```diff
@@ -97,5 +90,39 @@ public void Load(NodeSettings nodeSettings)
 
             this.callback?.Invoke(this);
         }
+
+        /// <summary>
+        /// Determines the default API port.
+        /// </summary>
+        /// <param name="network">The network to use.</param>
+        /// <returns>The default API port.</returns>
+        private static int DefaultPort(Network network)
+        {
+            int port;
```

Comment:
you can do without the variable and just `return network.IsTest() ? TestBitcoinApiPort : DefaultBitcoinApiPort;` etc

Reply:
Done.